spring-data-rest-webmvc/src/main/java/org/springframework/data/rest/webmvc/RepositorySchemaController.java

```diff
@@ -31,24 +34,38 @@
  * 
  * @author Jon Brisbin
  * @author Oliver Gierke
+ * @author Greg Turnquist
+ * @see http://json-schema.org/
  */
-@RepositoryRestController
```

Comment:
Let's roll this one back to make sure the controller is handled by the very last handler mapping to not cause any issues when overriding.